src/spec/page-objects/settings-page.js

```diff
@@ -9,6 +9,7 @@ class SettingsPage {
     this._domainInputButton = $('.domain--default-container button');
     this._domainsSetDefaultButtonList = element.all(by.css('.domain--default-action'));
     this._defaultDomain = $('.domain--default-container .default-domain');
+    this._domainsDisabled = element.all(by.css('.domain--disabled-action'));

```

Comment:
No, sorry, is a collection of buttons/links? what do you think about `_disableDomainButtons`?